backend/routes/issue.js

```diff
@@ -460,9 +460,35 @@ function updateIssue(req, res, next) {
 	});
 }
 
+
+function convertPoint(point) {
+	return [point[0], point[2], -point[1]];
+}
+
+function convertViewpoint(viewpoint) {
+	const pointsKeys = ['right', 'view_dir', 'look_at', 'position', 'up'];
+	pointsKeys.forEach((key) => {
+		if (viewpoint[key] && viewpoint[key].length === 3){
+			viewpoint[key] = convertPoint(viewpoint[key]);
+		}
+	});
+	return viewpoint;
+}
+
+	
+function sortViewpointAxis(issues) {
+	issues.forEach((issue) => {
+		convertViewpoint(issue.viewpoint);
```

Comment:
It's more than viewpoints. Pins are positions on the root object. (i.e `issue.position`)

You'd also have to convert the clipping plane normal.

Reply:
I now loop through the root object and then the viewpoint object , ensuring all points are addressed, including the clipping plane normal. :)